openvoxserver/Containerfile

```diff
@@ -76,28 +78,27 @@ COPY docker-entrypoint.d /docker-entrypoint.d
 COPY --from=builder /var/lib/gems/ /var/lib/gems/
 COPY --from=builder /usr/local/bin/r10k /usr/local/bin/
 
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-server/${OPENVOXSERVER_VERSION}/openvox-server_${OPENVOXSERVER_VERSION}-1%2Bubuntu24.04_all.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvox-agent/${OPENVOXAGENT_VERSION}/openvox-agent_${OPENVOXAGENT_VERSION}-1%2Bubuntu24.04_${TARGETARCH}.deb /
-ADD https://s3.osuosl.org/puppet-artifacts/openvoxdb/${OPENVOXDB_VERSION}/openvoxdb-termini_${OPENVOXDB_VERSION}-1%2Bubuntu24.04_all.deb /
+ADD https://apt.overlookinfratech.com/openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb /
+RUN apt-get update && \
+    apt-get install -y ca-certificates && \
+    dpkg -i /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb && \
+    rm /openvox${OPENVOX_RELEASE}-release-ubuntu${UBUNTU_VERSION}.deb
```

Comment:
If you have this in 2 commands, won't this end up in a layer that is still shipped to the end user?

Reply:
yes. this was more like a test if i can somehow cache this layer in build time... but it does not.